pom.xml

```diff
@@ -140,6 +140,12 @@
                 <module>tests-spark310+</module>
             </modules>
         </profile>
+        <profile>
+            <id>spark320tests</id>
+            <modules>
+                <module>tests-spark310+</module>
```

Comment:
so I think we need more changes in the tests-spark310+/pom.xml file to change the spark version based on the profile. Otherwise it will always use spark 3.1.0

Reply:
thanks for catching this. i consolidated version profiles into parent pom, I think it would be simpler. Not a maven expert but it seems to work as expected.